src/js/modules/infragistics.ui.scroll.js

```diff
@@ -557,7 +557,7 @@
 		_refreshScrollbarsDrag: function () {
 			if (this.options.scrollbarType === "custom" && this._vBarTrack && this._vBarDrag) {
 				this._vBarContainer.css("height", (this._elemHeight - 15) + "px");
-				this._vDragHeight = this._elemHeight * ((this._elemHeight - 2 * 15) / this._contentHeight);
+				this._vDragHeight = (this._elemHeight - 3 * 15) * this._percentInViewV;
```

Comment:
What is "3" and what is "15". Magic numbers. Please make them constants or variables.